projects/storefrontapp-e2e-cypress/cypress/helpers/product-configurator-cart.ts

```diff
@@ -24,19 +24,23 @@ export function clickOnEditConfigurationLink(cartItemIndex: number): void {
 }
 
 /**
- * Clicks on the 'Display Configuration' link in cart/order or quote for a certain item.
+ * Clicks on the 'Display Configuration' link in cart for a certain cart item.
  *
  * @param {number} cartItemIndex - Index of cart item
  */
 export function clickOnDisplayConfigurationLink(cartItemIndex: number): void {
-  locateCartConfiguratorElement(cartItemIndex);
+  cy.get('cx-cart-item-list .cx-item-list-row')
```

Comment:
Just out of curiosity: Why was this change needed?

Reply:
Good point! I rolled it back. The `locateCartConfiguratorElement(cartItemIndex);` method should be used